docs/Dockerfile

```diff
@@ -1,9 +1,9 @@
 ARG PYTHON_VERSION=3.11.0a7-slim
 
-FROM python:${PYTHON_VERSION} as builder
+FROM python:3.7-alpine3.12 as builder
```

Comment:
maybe update the line number #1 with the image?

Reply:
I will just remove it. Unnecessary ARG if you ask me